### ISSUE TITLE
CurrentLayoutIcon isn't Python 3 compatible

### ISSUE THREAD

**Issue by ramnes** [qtile/qtile#982]
```
Traceback (most recent call last):
  [...]
  File "/usr/lib64/python3.4/site-packages/libqtile/widget/currentlayout.py", line 123, in _configure
    self._setup_images()
  File "/usr/lib64/python3.4/site-packages/libqtile/widget/currentlayout.py", line 197, in _setup_images
    for layout_name in self._get_layout_names():
  File "/usr/lib64/python3.4/site-packages/libqtile/widget/currentlayout.py", line 167, in _get_layout_names
    in map(lambda x: (getattr(layout_module, x), x), dir(layout_module))
  File "/usr/lib64/python3.4/site-packages/libqtile/widget/currentlayout.py", line 168, in <listcomp>
    if isinstance(layout_class, (type, types.ClassType)) and issubclass(layout_class, Layout)
AttributeError: 'module' object has no attribute 'ClassType'
```

**Comment by and3rson** [comment]
Thanks for the report! I'm gonna fix this when I get to my PC.

**Comment by and3rson** [comment]
Fixed in #983

**Comment by flacjacket** [comment]
Done via #983
